src/routes/articlesRoutes.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const Article = require('../models/Article');
+const Articles = require('../models/Article');
```

Comment:
models/Article already required

Reply:
Both should be there, otherwise the code breaks